bitnami/mysql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mysql
-version: 6.13.2
+version: 6.14.0
```

Comment:
Do you think this change deserves bumping the version as minor instead of using patch?